compiler/next/lib/frontend/Parser/.gitignore

```diff
@@ -0,0 +1,4 @@
+bison-chapel.cpp
+bison-chapel.h
+flex-chapel.cpp
+flex-chapel.h
```

Comment:
We'll start checking in the parser in a while (probably a few weeks). We can have this until then if you like.